ci-operator/step-registry/baremetalds/assisted/operator/gather/baremetalds-assisted-operator-gather-commands.sh

```diff
@@ -15,8 +15,6 @@ fi
 # shellcheck source=/dev/null
 source "${SHARED_DIR}/packet-conf.sh"
 
-tar -czf - . | ssh "${SSHOPTS[@]}" "root@${IP}" "cat > /root/assisted-service.tar.gz"
```

Comment:
each step should be independent from the previous ones, at least on the technical level
so you'll have to handle that some way or another...

Reply:
🤔 This is copying the files from assisted-service repo to the remote server. Instead it should be the files from assisted-test-infra right?